src/pipecat/services/heygen/client.py

```diff
@@ -362,7 +362,7 @@ async def _write_audio_sleep(self):
         """Simulate audio playback timing with appropriate delays."""
         # Only sleep after we've sent the first second of audio
         # This appears to reduce the latency to receive the answer from HeyGen
-        if self._audio_seconds_sent < 1.0:
+        if self._audio_seconds_sent < 3.0:
```

Comment:
Sending a larger buffer before starting to wait, to avoid overflowing HeyGen with too many messages, has improved latency by around 600–700 ms.